build-system-tests/templates/components/react/cra/App.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from 'react';
+import { Amplify, Notifications } from 'aws-amplify';
+import {
+  AccountSettings,
+  Authenticator,
+  FileUploader,
+  MapView,
+  Text,
+  InAppMessagingProvider,
+  InAppMessageDisplay,
+} from '@aws-amplify/ui-react';
+import { StorageManager } from '@aws-amplify/ui-react-storage';
+import '@aws-amplify/ui-react/styles.css';
+import awsconfig from './aws-exports';
+Amplify.configure(awsconfig);
+
+const { InAppMessaging } = Notifications;
+
+export default function Home() {
+  useEffect(() => {
+    // sync remote in-app messages
+    InAppMessaging.syncMessages();
+  }, []);
+
+  return (
+    <>
+      <InAppMessagingProvider>
+        <InAppMessageDisplay />
+        <Text>In-App Messaging Example</Text>
+      </InAppMessagingProvider>
+      <FileUploader acceptedFileTypes={['image/*']} accessLevel="public" />
+      <Authenticator>
+        {({ signOut, user = { username: '' } }) => (
+          <main>
+            <h1>Hello {user.username}</h1>
```

Comment:
What about 
```suggestion
        {({ signOut, user }) => (
          <main>
            <h1>Hello {user.username}</h1>
```
to mimic our documentation code?

Reply:
or `user?.username` if above isn't TS compliant.

Reply:
Let's add a question mark for now. Let's add a ticket for updating our docs for TS strict customers.

Reply:
THe reason why I didn't use a question mark was because that way I can re-use the App.js file for both JS and TS.

Do you prefer to re-use the file or make another file for TS?